keps/sig-instrumentation/647-apiserver-tracing/README.md

```diff
@@ -207,15 +202,19 @@ Alpha
 
 Beta
 
-- [] Tracing 100% of requests does not break scalability tests (this does not necessarily mean trace backends can handle all the data).
+- [X] Tracing 100% of requests does not break scalability tests (this does not necessarily mean trace backends can handle all the data).
```

Comment:
Couple smaller comments:

1) For testing section:
- can you please add the coverage numbers (as in the template)
- for integration tests - can you please link to k8s-triage (as described in the template)

2) for beta criteria:

>  Tracing 100% of requests does not break scalability tests (this does not necessarily mean trace backends can handle all the data).

For my own education - how did you do that? I think we didn't modify the periodic tests - should we?


3) SLIs question - are there any plans to have it addressed on OpenTelemetry?

4) Scalability section - there is a new question to answer:
https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#can-enabling--using-this-feature-result-in-resource-exhaustion-of-some-node-resources-pids-sockets-inodes-etc

Reply:
It was a one-time test with tracing set at 100%: https://github.com/kubernetes/kubernetes/pull/113695#issuecomment-1307665358. It isn't continuous.

Reply:
For SLIs, some in OTel are working on proposals: https://github.com/open-telemetry/oteps/pull/238.  I don't know when it will land, though.

Reply:
Addressed points 1 and 4 in https://github.com/kubernetes/enhancements/pull/4333/commits/4500ad8cf027bca5e05ee138746643868f8ade1f

Reply:
Done in https://github.com/kubernetes/enhancements/pull/4333/commits/47c37e437cecb2d646ae4619485a5a3deac4c056